package.yaml

```diff
@@ -77,7 +79,7 @@ default-extensions:
 
 dependencies:
   - aeson
-  - base
+  - base < 5
```

Comment:
Interesting. Why below 5?

Reply:
:man_shrugging: 
`stack sdist` was yelling at me that I had to specify an upper limit to the `base` version so I just said 5 since we're on 4.12.0 right now

Reply:
Great 👍